tests/jballerina-integration-test/src/test/resources/packaging/dynamic_listener_async_call_test_project/main.bal

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) 2022 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Lets add test with multiple module register stop handlers, register same stop handler multiple times and register stop handler inside functions.